src/platformio.ini

```diff
@@ -18,6 +18,7 @@ extra_configs =
 	targets/namimnorc_900.ini
 	targets/namimnorc_2400.ini
 	targets/neutronrc_900.ini
+	targets/newbeedrone_2400.ini
```

Comment:
We don't actually need these changes any more.

Reply:
We do unless unified configurator is released first before this is merged.

Reply:
We really only need an alias in Configurator because it's the same as any other generic target!